shimmy/meltingpot_compatibility.py

```diff
@@ -5,20 +5,20 @@
 and modified to modern pettingzoo API
 """
 # pyright: reportOptionalSubscript=false
-
+# isort: skip_file
```

Comment:
Im still confused about this, if I understood correctly, locally the change in this file revert while on CI this raises an error?

Reply:
Yeah I have no idea what caused it but CI throws an error whereas my local pre-commit hook pass, and if I revert it to what the CI wants then my local hooks fail and revert it. Actually I guess what I could do is remove this part and then just remove my local pre-commit hooks temporarily and then push it without these isort skip file stuff.

Reply:
Removed this